lambdas/transcode/index.py

```diff
@@ -0,0 +1 @@
+from transcode.index import lambda_handler
```

Comment:
I had to read this twice then I realized what it was.
```suggestion
"""entrypoint for lambda at root"""


from transcode.index import lambda_handler
```

Reply:
Haha, I got confused by this myself just now. It's actually not needed if we're not packaging ffmpeg here, so I'll remove it.